tests/test_cli_core.py

```
@@ -0,0 +1,1402 @@
+class TestInitTemplateErrors:
+    """Test template rendering error handling in init."""
+
+    def test_init_template_render_failure(self, tmp_path, capsys):
+        """init should handle template rendering errors."""
+        os.chdir(tmp_path)
+        (tmp_path / ".git").mkdir()
+
+        # Mock shutil.copy to fail during template copy
+        original_copy = shutil.copy
+
+        def failing_copy(src, dst):
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `failing_copy`

(ANN202)